### DISCUSSION TITLE
Add githubLatestTag function?

### DISCUSSION THREAD

**Discussion by danielo515** [twpayne/chezmoi#2266] [Q&A] [answered]
Hello.
I just discovered the function githubLatestRelease, it is quite neat. However, some repositories do not have releases, and therefore that function will fail, but what they have are tags. What about creating a new command called githubLatestTag to get the latest available tag?

**Comment by twpayne** [comment, marked as answer]
Makes sense. I've created #2267 to add the functionality.

Note that the GitHub API does not provide an easy way to find the latest tag (you have to list all tags, and then inspect the date of each tag's commit) but when listing tags GitHub *seems* to return the latest tag first.

**Comment by danielo515** [comment]
That was quick, awesome thanks.

**Reply by twpayne** [reply]
@danielo515 did the addition of the [`gitHubLatestTag` function](https://www.chezmoi.io/reference/templates/github-functions/gitHubLatestTag/) answer your question?

**Comment by danielo515** [comment]
Yes, absolutely

On Fri, 2 Sep 2022 at 02:59, Tom Payne ***@***.***> wrote:

> @danielo515 <https://github.com/danielo515> did the addition of the
> gitHubLatestTag function
> <https://www.chezmoi.io/reference/templates/github-functions/gitHubLatestTag/>
> answer your question?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/twpayne/chezmoi/discussions/2266#discussioncomment-3531781>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARKJWO6FNSMVLBRGLDUQL3V4FGPDANCNFSM564VT25A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
---
https://danielorodriguez.com
